### ISSUE TITLE
Development-Future & Multi Pane?

### ISSUE THREAD

**Issue by 0pLuS0** [rodlie/qtfm#180]
Hello,

Remember me? LOL, it's been a long time, hopefully you remember me.

Well I'm back, because Slackware 15.0 was just released on 02/03/22!

And with all the great changes, well, SpaceFM has been a long dead project, I've been waiting patiently on an alternative, and you're the best I've seen so far, and easier to get a hold of and discuss bugs, etc...

So I wanted to find out how the development/future is looking for the project? I'm hoping it's going to be around for a long time.

I also asked in the past about making qtfm Multi Pane like SpaceFM, here I'm using 3 Panes.

![multi_panes](https://user-images.githubusercontent.com/36190223/158738189-adb7669e-0b86-4ddd-b2a7-544005edde88.jpg)

I really want/need to have a file manager with several panes, because of the amount of things I do, it makes it easier moving around in the system this way.

I also tried your Dark Mode, but it's doesn't work correctly, the left side pane, there is no text on both the main ui, and in the settings ui too.

It would be nice if the panes on the left side appeared also in dark and then the text in white.

BUT, I personally don't think you need this Dark Mode and should just remove it, because anyone can easily theme QT apps.

![1 dark_theme](https://user-images.githubusercontent.com/36190223/158738336-bdfe10ec-1804-4b0f-aeac-24ce83841b46.jpg)

![2 dark_theme](https://user-images.githubusercontent.com/36190223/158738398-5bf1dd66-2b8a-4a61-bae0-e68a5461353c.jpg)

**Comment by 0pLuS0** [comment]
The trash can on the left side pane just disappeared on me.

I was testing to see how it worked, I made a folder and file and then moved them to the trash can, I then clicked on the trash, and started to right click on it to empty it and it disappeared.

Hmm

Notice there is no longer a trash can on the left side pane.

![no_trash](https://user-images.githubusercontent.com/36190223/158739603-ec842a8b-2091-4a78-95e2-6df69c671a73.jpg)

**Comment by 0pLuS0** [comment]
I noticed when I right click and pick ```new folder``` or ``` new file``` when they first appear in qtfm, the placement of the folder or file is offset to the right, it's not centered under the word(s). Then when I click on qtfm anywhere, the folder/file center themselves.

Here's a folder when I first created it, notice it's off center, more to the right of the text.

![alignment](https://user-images.githubusercontent.com/36190223/158740009-8ff86b77-8b85-42f5-b5cc-1f47d4f7d036.jpg)

**Comment by 0pLuS0** [comment]
I'm using an app called qt5gtk to change fonts/themes/icons of QT apps;

https://bitbucket.org/trialuser02/qt5gtk2/src/master/

When I pick a font for a QT app, only the Menus and Settings UI change. The fonts for the Main UI and Pane don't change their fonts.

**Comment by fcladera** [comment]
Hello, thanks for this amazing project!

Are there any updates on the multi-pane feature? 

Thanks!

**Comment by 0pLuS0** [comment]
Hmm I guess the project is dead for the moment.

rodlie was very active last year.

So sad, spacefm is dead too, we really need some active development for a Multi Pane FM in Linux.

**Comment by edmundlaugasson** [comment]
I agree with these feature requests. Besides, there is multipane,  -tabbed file manager [Krusader](https://krusader.org/) , based on Qt.
Also based on GTK, there is [DoubleCMD](https://doublecmd.sourceforge.io/) .

**Comment by 0pLuS0** [comment]
I'm still using SpaceFM, still haven't found anything as modern looking...
